gluefactory/models/extractors/joint_point_line_extractor.py

```diff
@@ -212,6 +215,23 @@ def _init(self, conf):
         self.line_extractor = LineExtractor(
             self.conf.line_detection.conf,
         )
+        deeplsd_conf = {
```

Comment:
We should add a config option to activate/deactivate use of deeplsd here! Because loading in additional model anytime hurts memory and performance in case we just train our model or do non-line related tasksm on it.

Reply:
Agreed, will implement that